db/view/view.hh

```diff
@@ -123,6 +123,28 @@ bool matches_view_filter(const schema& base, const view_info& view, const partit
 
 bool clustering_prefix_matches(const schema& base, const partition_key& key, const clustering_key_prefix& ck);
 
+struct bytes_with_action {
```

Comment:
This also appears unrelated.

Reply:
I have to be honest I didn't understand what this struct is about when encountering it. It mixes a few seemingly unrelated concept, and called with the very general-sounding term "action".